OpenSSL/SSL.py

```diff
@@ -1883,6 +1883,18 @@ def get_cipher_version(self):
             return version.decode("utf-8")
 
 
+    def get_protocol_version(self):
+        """
+        Obtain the protocol version of the current connection.
+
+        :returns: The TLS version of the current connection, for example
+            the value for TLS 1.2 would be 0x303.
+        :rtype: :py:class:`unicode`
+        """
+        version = _ffi.string(_lib.SSL_get_version(self._ssl))
+        return version.decode("utf-8")
```

Comment:
For the first paragraph; what I’m saying is that the docstring is plain wrong.  There’s no way you get anything resembling “0x303” back.  You get a string like `"TLSv1"` back.  Try it and pdb in your test. :)